lib/react.js

```diff
@@ -196,11 +195,19 @@ declare module react {
   ) : void;
 
   declare export var createClass: React$CreateClass;
+  declare export function createContext<T>(
+    defaultValue: T,
+  ): {
+    Provider: React$ComponentType<{value: T}>,
+    Consumer: React$ComponentType<{children: (value: T) => React$Node}>,
+  };
   declare export var createElement: React$CreateElement;
   declare export var cloneElement: React$CloneElement;
   declare export function createFactory<ElementType: React$ElementType>(
     type: ElementType,
   ): React$ElementFactory<ElementType>;
+  declare export function createRef<ElementType: React$ElementType>(
+  ): {value: null | React$ElementRef<ElementType>};
```

Comment:
Need to change this to `current`.

Reply:
Is there any difference between `null | React$ElementRef<ElementType>` and `?React$ElementRef<ElementType>` ?

Reply:
The latter includes `undefined`. For basic code questions please ask on Stack Overflow or another site so we can keep this uncluttered and avoid spamming people.